controllers/admin/AdminCountriesController.php

```diff
@@ -86,12 +86,12 @@ public function __construct()
                 'filter_key' => 'b!name'
             ),
             'iso_code' => array(
-                'title' => $this->l('ISO code'),
+                'title' => $this->trans('ISO code', array(), 'Admin.Global'),
```

Comment:
You can use Admin.International.Feature, it can be found on several pages, but all are from the International section.